packages/server/src/main/java/software/uncharted/terarium/hmiserver/proxies/documentservice/ExtractionProxy.java

```diff
@@ -1,28 +1,57 @@
 package software.uncharted.terarium.hmiserver.proxies.documentservice;
 
+import java.util.UUID;
 
 import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
 import software.uncharted.terarium.hmiserver.models.documentservice.autocomplete.AutoComplete;
 import software.uncharted.terarium.hmiserver.models.documentservice.responses.XDDExtractionsResponseOK;
 import software.uncharted.terarium.hmiserver.models.documentservice.responses.XDDResponse;
 
-@FeignClient(name = "extractionService", url = "${xdd-dev-service.url}")
+@FeignClient(name = "extractionService", url = "${xdd-prod-service.url}")
```

Comment:
Is this intentional/coordinated with the XDD team? I know that a number of features/docs have previously been unavailable in their prod instance.

Reply:
To add to this, I think that the xdd documents controller is still on dev not prod, which can cause inconsistencies between extractions and docs

Reply:
If i hit dev I get a 413 request too large. Only prod currently works. Not sure what we want to do here.

Reply:
We should at least keep the extractions and docs endpoints in sync. Reaching out to Ian at xDD may help?

Reply:
I'll revert back to dev so this PR doesn't break anything else. And we can debug why the dev endpoint for cosmos extractions is behaving wonky separately